app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,7 @@
             android:name=".view.profile.EditProfileActivity"
             android:windowSoftInputMode="adjustPan" />
         <activity android:name=".view.profile.ProfileActivity" />
-        <activity android:name=".view.activity.UploadActivityActivity" />
+        <activity android:name=".view.activity.UploadActivityActivity" android:windowSoftInputMode="adjustNothing"/>
```

Comment:
what does this windowSoftInputMode ?

Reply:
In fact, as the view is sort of divided in two with a scroll part and a fixed part, opening the keyboard pushed the fixed part reducing the scroll. This parameter allows the keyboard to appear on top of the layout instead of pushing it from below.